app/views/qsets/_qset_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row qset_list">
+  <% @qsets.each_with_index do |qset, index| %>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <div class="row valign">
```

Comment:
This `<div>` can be removed; the classes can be moved into the `<h4>` element above.